### PR TITLE
MRGME: MD

### PR DESCRIPTION
**Pull request by tlyon3**
Implementation for Maryland data

### CODE REVIEW

**Review by cc7768 — APPROVED** [review]
I made a few changes and refactors but this looked good.

@tlyon3 if you'll look over these changes and, if you don't have any questions, it should be ready to merge

File: src/cmdc_tools/datasets/__init__.py [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-    CANStateTimeseries,
-)
+from .cex import CountyDex, DailyCountyLex, DailyStateLex, StateDex
+from .covidactnow import (CANCountyActuals, CANCountyTimeseries,
+                          CANStateActuals, CANStateTimeseries)
 from .covidtrackingproject import CTP
+from .db_util import TempTable, fast_to_sql
 from .dol import StateUIClaims
+from .jhu import DailyReports, DailyReportsUS, Locations
+from .official import LA, Alaska, CACountyData, Imperial, Maryland, SanDiego
```
Comment (tlyon3) [inline src/cmdc_tools/datasets/__init__.py]: I think this needs to import `Arkansas` and `MA` as well

File: src/cmdc_tools/datasets/official/AR/data.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            params = {
+                "f": "json",
+                "where": "1=1",
+                "outFields": "*",
+                "returnGeometry": "false",
+            }
+
+        super(Arkansas, self).__init__(params)
+
+    def _insert_query(self, df, table_name, temp_name, pk):
```
Comment (tlyon3) [inline src/cmdc_tools/datasets/official/AR/data.py:23]: @cc7768 I'm guessing that this is using the most recent code for `Arkansas` from the master branch? That branch was merged into Master, correct? If this is rebased with the most recent Master, this is good to go.

**Review by tlyon3 — COMMENTED** [review]
@cc7768 A couple questions regarding these changes

File: src/cmdc_tools/datasets/__init__.py (hunk 1 above)
Comment (cc7768) [reply src/cmdc_tools/datasets/__init__.py]: Good catch -- Now they're imported.

File: src/cmdc_tools/datasets/official/AR/data.py (hunk 2 above)
Comment (cc7768) [reply src/cmdc_tools/datasets/official/AR/data.py:23]: Yep. Everything is on most recent master.

**Pull request merged**